mmv1/products/compute/api.yaml

```diff
@@ -1672,6 +1672,32 @@ objects:
         name: 'securityPolicy'
         description: |
           The security policy associated with this backend service.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'securitySettings'
+        description: |
+          The security settings that apply to this backend service. This field is applicable to either
+          a regional backend service with the service_protocol set to HTTP, HTTPS, or HTTP2, and
+          load_balancing_scheme set to INTERNAL_MANAGED; or a global backend service with the
+          load_balancing_scheme set to INTERNAL_SELF_MANAGED.
+        properties:
+          # TODO: 'Region' resource type is incorrect and should be ClientTLSPolicy
```

Comment:
Can you move this comment from its own line to right after L1686? I think it's messing up the YAML parser.

Reply:
done